pkg/ocibundle/tools/oci.go

```diff
@@ -0,0 +1,55 @@
+package tools
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"syscall"
+
+	"github.com/opencontainers/runtime-spec/specs-go"
+
+	"github.com/opencontainers/runtime-tools/generate"
+)
+
+// RootFs is the default root path for OCI bundle
+type RootFs string
```

Comment:
why need a separate type?

Reply:
A convenient way to append `rootfs` for a bundle